src/components/Map/index.js

```diff
@@ -59,6 +72,19 @@ const LeaftletMap = ({
           <MarkerWrapper position={coordinates} onMarkerSet={onWrapperMarkerSet} />
         </>
       ) : null}
+      <SearchControl
+        provider={provider}
+        showMarker={false}
+        showPopup={false}
+        retainZoomLevel={false}
+        animateZoom={true}
+        autoClose={false}
+        searchLabel={'Enter address, please'}
```

Comment:
this needs to be translated via `react-intl` and the `useIntl` hook (or maybe a `FormattedMessage` will also work)